pkg/auth/signing/marshaling_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package signing_test
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/pkg/pb"
+)
+
+func TestOrderSerialization(t *testing.T) {
+	t.Skip("broken")
```

Comment:
😢

Reply:
Yeah, it needs separate fixes to the serialization to make it compatible. It was already broken before the current PR.

Reply:
But does that mean the serialization has already changed and it might change again? That would be not good.